pkg/loop/internal/test/logger_loop.go

```diff
@@ -20,15 +20,32 @@ const LoggerTestName = "server-side-logger-name"
 // as well as the test at `./pkg/loop/logger_loop_test.go`
 type GRPCPluginLoggerTest struct {
 	plugin.NetRPCUnsupportedPlugin
-
-	logger.Logger
+	logger.SugaredLogger
+	ErrorType int
 }
 
 func (g *GRPCPluginLoggerTest) GRPCServer(*plugin.GRPCBroker, *grpc.Server) (err error) {
-	err = errors.New("test error")
-	g.Logger.Errorw("Error!", "err", err)
-	err = errors.Join(err, g.Logger.Sync())
-	time.Sleep(time.Second)
+	//Simulate panic/error/log after GRPC is started, if a panic is thrown before the GRPC server is initialized
+	//it will not be caught as stderr will be closed before HashiCorp plugin will have a change to read from it
+	go func() {
+		time.Sleep(time.Second)
+		switch g.ErrorType {
+		case 0:
```

Comment:
Maybe worth an enum so these can be named?

Reply:
Added in [`11a37f3` (#384)](https://github.com/smartcontractkit/chainlink-common/pull/384/commits/11a37f37be63708efaad9535f23a70d4d848e584)